esp/services/ws_fs/ws_fsBinding.cpp

```diff
@@ -386,6 +387,23 @@ void CFileSpraySoapBindingEx::xsltTransform(const char* xml, const char* sheet,
 }
 #endif
 
+static void checkDropZoneInputAndAccess(IEspContext &context, StringBuffer& dropZoneName, const char* host, const char* path, SecAccessFlags accessReq)
```

Comment:
minor: dropZoneName isn't used by any of the callers, so it would be clearer if it were a const char *

Reply:
See other comments - looks like this method should be used by ws_fsService.cpp too.

Reply:
If this method is used by ws_fsService.cpp, the StringBuffer& dropZoneName is needed and the method should return the dropzone tree.